README.md

```diff
@@ -1,7 +1,88 @@
+<a href="https://github.com/"><img alt="https://senior-knights.github.io/course-schedulizer/" src="https://img.shields.io/badge/github-000?logo=github" /></a>
+<a href="https://www.npmjs.com/"><img alt="https://senior-knights.github.io/course-schedulizer/" src="https://img.shields.io/badge/npm-000?logo=npm" /></a>
+<a href="https://www.netlify.com/"><img alt="https://senior-knights.github.io/course-schedulizer/" src="https://img.shields.io/badge/netlify-000?logo=netlify" /></a>
+
+## Background
+
+Every year, all department chairs at Calvin must develop a schedule for their department’s classes based on a spreadsheet provided to them by the Registrar. The schedule must contain the times, professors, and rooms for every class section in the department and the schedule must satisfy the following constraints: pnly one section can be in a room at a time, a professor can only teach one section at a time, among others. These constraints make it extremely difficult for the department chair to create a schedule without a tool to help them. We propose building a web application, named the Course Schedulizer, that will allow department chairs to visualize and manipulate their department course schedules. It will provide: the ability to upload and export department schedules via CSV, integration with the spreadsheets provided and required by the Registrar, two views to visualize the schedule data (by location and by instructor), schedule conflict detection and resolution suggestions (for honors portion), and many others.
+
+For more information, please visit the [About Page](https://senior-knights.github.io/course-schedulizer/#/about).
+
+## How to use
+
+Detailed information can be found on our [Help Page](https://senior-knights.github.io/course-schedulizer/#/help) (once we make it).Access our production website and upload a CSV following the prescribed specifications:
+
+- Department: string
+- Term: [0-9][0-9]/(FA | SP | IN)
+- TermStart: mm/dd/yyyy
+- AcademicYear: yyyy
+- SectionName: SubjectCode-CourseNum-SectionCode
+- SubjectCode: string
+- CourseNum: string
+- SectionCode: string
+- CourseLevelCode: pos num
+- MinimumCredits: pos num
+- FacultyLoad: pos num
+- Used: pos num
+- Day10Used: pos num
+- LocalMax: pos num
+- Global Max: pos num
+- RoomCapacity: pos num
+- BuildingAndRoom: string
+- MeetingDays: M?T?W?(TH)?(F)
+- MeetingTime: xx:xx(AM | PM) - xx:xx(AM | PM)
+- SectionStartDate: mm/dd/yyyy
+- SectionEndDate: mm/dd/yyyy
+- Building: string
+- RoomNumber: string
+- MeetingStart: xx:xx(AM | PM)
+- MeetingStartInternal: xx:xx:xx 24-hour
+- MeetingEnd: xx:xx(AM | PM)
+- MeetingEndInternal: xx:xx:xx 24-hour
+- Monday: "M" or ""
+- Tuesday: "T" or ""
+- Wednesday: "W" or ""
+- Thursday: "TH" or ""
+- Friday: "F" or ""
+- ShortTitle: string
+- Faculty: string (first and last)
+- SectionStatus: string
+- InstructionalMethod: "LEC" | "CPI" | "IND" | "TUT" | "SEM"
```

Comment:
This is the change I'm least confident about, so if you don't like this idea feel free to ignore it, but I think it might be a helpful reference to have examples like this.

```suggestion
- Department: string (like `Mathematics`)
- Term: [0-9][0-9]/(FA | SP | IN) (like `21/SP` for Spring 2021)
- TermStart: mm/dd/yyyy (like `3/29/2021` or `12/1/2022`)
- AcademicYear: yyyy (like `2021`)
- SectionName: SubjectCode-CourseNum-SectionCode (like `MATH-252-B`)
- SubjectCode: string (like `MATH`)
- CourseNum: string (like `252` or `252L` for a lab)
- SectionCode: string (like `B`)
- CourseLevelCode: pos num (like `200` for a 200 level course)
- MinimumCredits: pos num (like `3` or `3.5`)
- FacultyLoad: pos num (like `4` or `4.5`)
- Used: pos num (like `20`)
- Day10Used: pos num (like `22`)
- LocalMax: pos num (like `25`)
- GlobalMax: pos num (like `30`)
- RoomCapacity: pos num (like `32`)
- BuildingAndRoom: string (like `HH 345`)
- MeetingDays: M?T?W?(TH)?F? (like `MWTHF`)
- MeetingTime: xx:xx(AM | PM) - xx:xx(AM | PM) (like `9:00AM - 9:50AM`)
- SectionStartDate: mm/dd/yyyy (like `3/29/2021` or `12/1/2022`)
- SectionEndDate: mm/dd/yyyy (like `3/29/2021` or `12/1/2022`)
- Building: string (like `HH`)
- RoomNumber: string (like `345`)
- MeetingStart: xx:xx(AM | PM) (like `2:30PM`)
- MeetingStartInternal: xx:xx:xx 24-hour (like `14:30:00`)
- MeetingEnd: xx:xx(AM | PM) (like `3:20PM`)
- MeetingEndInternal: xx:xx:xx 24-hour (like `13:20:00`)
- Monday: `M` or empty
- Tuesday: `T` or empty
- Wednesday: `W` or empty
- Thursday: `TH` or empty
- Friday: `F` or empty
- ShortTitle: string (like `Number Theory`)
- Faculty: string (first and last) (like `Paul Erdos`)
- SectionStatus: string (like `Active`)
- InstructionalMethod: `LEC`, `CPI`, `IND`, `TUT`, or `SEM`
```